webservices/server-integration/src/main/java/org/jboss/as/webservices/logging/WSLogger.java

```diff
@@ -225,19 +225,9 @@ public interface WSLogger extends BasicLogger {
     @Message(id = 52, value = "Unsupported handler chain type: %s. Supported types are either %s or %s")
     StartException wrongHandlerChainType(String unknownChainType, String knownChainType1, String knownChainType2);
 
-//    @Message(id = 53, value = "Cannot add new handler chain of type %s with id %s. This id is already used in config %s for another chain.")
-//    StartException multipleHandlerChainsWithSameId(String chainType, String handlerChainId, String configId);
-
     @Message(id = 54, value = "Config %s: %s handler chain with id %s doesn't exist")
     OperationFailedException missingHandlerChain(String configName, String handlerChainType, String handlerChainId);
 
-    // @Message(id = 55, value = "Config %s, %s handler chain %s: doesn't contain handler with name %s")
-    // OperationFailedException missingHandler(String configName, String handlerChainType, String handlerChainId, String handlerName);
-
-    //@LogMessage(level = ERROR)
-    //@Message(id = 56, value = "Method invocation failed with exception: %s")
-    //void methodInvocationFailed(@Cause Throwable cause, String message);
```

Comment:
Please restore the commented methods. They provide documentation of what the message id was used for and help prevent reuse for other purposes by showing any gaps in the currently used id #s was intentional.